GX-TimeFrequency/src/parsers/parse_minimax.f90

```diff
@@ -85,11 +85,9 @@ subroutine read_freq_grid(npoints, file_name, erange, grid, weights)
           read(tempStr(2),*) lower_range
           read(tempStr(3),*) upper_range
           if(erange > lower_range.and.erange <= upper_range) then
-             do i =  1, 2*npoints
+             do i =  1, npoints
```

Comment:
Ah, this is because you've moved the weights onto the same line as grid points? Very nice